### PR TITLE
Add settings panel

### PR DESCRIPTION
**Pull request by ferjm**
I added a side panel and moved the workers select there. The panel is shown by hovering over the settings icon and hidden when not hovering over the panel.

### CODE REVIEW

**Comment by ferjm** [comment]
It looks like this:

screen

File: app/scripts/ui.js [hunk 1]
```diff
@@ -171,16 +171,14 @@ const UI = {
     this._cleanup();
   },
 
-  initWorkerSelect(runningWorkersElem) {
+  _initWorkersSelect(select) {
+    select.firstElementChild.value = DEFAULT_WORKERS;
```
Comment (CYBAI) [inline app/scripts/ui.js]: With this line, we'll make the default select as `DEFAULT_WORKERS` but I'm wondering, will it be better to show the default number to the users and let them know it's default with which number without selecting.

Maybe like
```js
select.firstElementChild.textContent = `Number of workers: (Default with ${DEFAULT_WORKERS})`;
```

Comment (ferjm) [reply app/scripts/ui.js]: Sounds good. I just changed. Thanks.

**Pull request merged**